test/dropdown.html

```diff
@@ -157,24 +156,6 @@
         datepicker.opened = false;
         expect(datepicker.focusElement.hasAttribute('focus-ring')).to.be.false;
       });
-
-      it('should not close on clear-button down', () => {
```

Comment:
I think these two tests should be kept and the behaviour needs to be fixed so that these still pass. Now the behaviour is not so nice anymore, when overlay always opens. See my comment here: https://github.com/vaadin/vaadin-date-picker/pull/648#issuecomment-476639634

Reply:
Latest version looks good now. Thanks!